service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NotifyGatekeeperTest.java

```diff
@@ -25,7 +25,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
```

Comment:
why were there changes to this file?

Reply:
it was updated following refactoring of `NotificationTemplates` in `NotificationTemplateType` an enum